ssn/chapters/Observation.html

```diff
@@ -450,14 +448,15 @@ <h5><a href="#SOSASensor">sosa:Sensor</a></h5>
         </p>
         <p>
             <em property="rdfs:label">Sensor</em> -
-            <span property="skos:definition">An identifiable entity that can generate <a
-                    href="#SOSAObservation">Observations</a> pertaining to an <a href="#SOSAProperty">Property</a> by
-                implementing a <a href="#SOSAProcedure">Procedure</a>.
-                Device, agent (including humans), or software (simulation)
-                involved in, or implementing, a <a href="#SOSAProcedure">Procedure</a>.
-                <a href="#SOSASensor">Sensors</a> respond to a
-                <a href="#SOSAStimulus">Stimulus</a>, e.g., a change in the environment,
-                or input data composed from the results of prior <a href="#SOSAObservation">Observations</a>
+            <span property="skos:definition">An entity that that implements a (Observation)
+                <a href="#SOSAProcedure">Procedure</a> in the context of an Observation in order to determine the value
+                of a (observable) <a href="#SOSAProperty">Property</a>.
```

Comment:
```suggestion
                of an (observable) <a href="#SOSAProperty">Property</a>.
```

Reply:
I guess I'm being too cute with the parentheses ;-)